src/Microsoft.TestPlatform.Client/TestPlatform.cs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.VisualStudio.TestPlatform.Client
     /// <summary>
     /// Implementation for TestPlatform
     /// </summary>
-    public class TestPlatform : ITestPlatform
+    internal class TestPlatform : ITestPlatform
```

Comment:
>internal [](start = 4, length = 8)

I am assuming adapters to customer havent taken a dependency on this. BTW

Reply:
[https://mseng.visualstudio.com/_search?type=code&lp=custom-Collection&text=ITestPlatform&_a=contents&result=DefaultCollection%2FVSOnline%2FVSO%2FGBmaster%2F%2FTfs%2FExtendedClient%2FTestPlatform%2FClient%2FUtilities.cs&filters=ProjectFilters%7BVSOnline%7D](https://mseng.visualstudio.com/_search?type=code&lp=custom-Collection&text=ITestPlatform&_a=contents&result=DefaultCollection%252FVSOnline%252FVSO%252FGBmaster%252F%252FTfs%252FExtendedClient%252FTestPlatform%252FClient%252FUtilities.cs&filters=ProjectFilters%257BVSOnline%257D)

---
In reply to: [177697101](https://github.com/Microsoft/vstest/pull/1517#discussion_r177697101) [](ancestors = 177697101)